sdk/webpubsub/Azure.Messaging.WebPubSub/api/Azure.Messaging.WebPubSub.netstandard2.0.cs

```diff
@@ -99,7 +99,7 @@ namespace Microsoft.Extensions.Azure
 {
     public static partial class WebPubSubServiceClientBuilderExtensions
     {
-        public static Azure.Core.Extensions.IAzureClientBuilder<Azure.Messaging.WebPubSub.WebPubSubServiceClient, Azure.Messaging.WebPubSub.WebPubSubServiceClientOptions> AddWebPubSubServiceClient<TBuilder>(this TBuilder builder, string connectionString, string hub) where TBuilder : Azure.Core.Extensions.IAzureClientFactoryBuilder { throw null; }
+        public static Azure.Core.Extensions.IAzureClientBuilder<Azure.Messaging.WebPubSub.WebPubSubServiceClient, Azure.Messaging.WebPubSub.WebPubSubServiceClientOptions> AddWebPubSubServiceClient<TBuilder>(this TBuilder builder, string endpoint, string hub) where TBuilder : Azure.Core.Extensions.IAzureClientFactoryBuilder { throw null; }
```

Comment:
This change is because we kept the same method that is generated and removed the customized one.
The `CodeGenSuppress` attribute cannot suppress a method with generic parameter, if this change is not acceptable, we will have to suppress the whole generated extension class and restore all the customized code as a temporary solution

Reply:
for now made webpubsub all custom to avoid the param name change until we can get the static class to adhere to existing type